testsuite/MDAnalysisTests/analysis/test_persistencelength.py

```diff
@@ -149,18 +149,29 @@ def test_sortbb(self, u):
 
         assert_equal(s_ag.ids, [0, 1, 4, 6, 8])
 
-    def test_not_fragment(self, u):
```

Comment:
Why did you remove the test? Did you replace it with another one?

Reply:
This test relied on the old logic based on `n_fragments`, which was changed here.